pages.de/common/nativefier.md

```diff
@@ -0,0 +1,16 @@
+# nativefier
+
+> Befehlszeilen-Tool zum Erstellen einer Desktop-Anwendung für jede Website mit minimaler Konfiguration.
+> Weitere Informationen <https://github.com/jiahaog/nativefier>.
+
+- Erstellen einer Desktop-Anwendung für eine Website:
+
+`nativefier {{URL}}}`
+
+- Erstellen Sie eine Desktop-Anwendung mit einem benutzerdefinierten Namen:
+
+`nativefier --Name {{Name}} {{URL}}`
```

Comment:
I think we have a casing issue with the argument here. The original page uses `--name`, but here we have `--Name`?